src/js/utils/refs.js

```diff
@@ -2,6 +2,6 @@ import { useImperativeHandle, useRef } from 'react';
 
 export const useForwardedRef = (ref) => {
   const innerRef = useRef(null);
-  useImperativeHandle(ref, () => innerRef.current);
+  useImperativeHandle(ref, () => innerRef.current, [innerRef]);
```

Comment:
I think this makes alot of sense, especially since what is returned is just the ref.

Reply:
👍